docs/guidance/stable-diffusion-rayservice.md

```diff
@@ -21,7 +21,7 @@ kubectl apply -f ray-service.stable-diffusion.yaml
 
 This RayService configuration contains some important settings:
 
-* Its `tolerations` for workers match the taints on the GPU node group. Without the tolerations, worker Pods won't be scheduled on GPU nodes.
+* Its `tolerations` for workers match the taints on the GPU node group (which has taints), so they can be scheduled on either GPU or CPU node. We don't add these to `headGroupSpec` to make sure head Pod & KubeRay operator Pod are not allocated to GPU node group (which has taints).
```

Comment:
The `tolerations` for workers allow them to be scheduled on nodes without any taints or on nodes with specific taints. However, workers will only be scheduled on GPU nodes because we set `nvidia.com/gpu: 1` in the Pod's resource configurations.